Fixed deprecated method usages of MessageManager

drpayyne opened the pull request:
### Description

- Fixed **all** deprecated method usages of MessageManager in `Magento_Wishlist`
- Fixed appropriate PHPUnit tests

### Manual testing scenarios

- ONLY code modifications

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

drpayyne committed:
Fixed deprecated method usages of MessageManager

drpayyne committed:
Fixed code style issues

ihor-sviziev commented on app/code/Magento/Catalog/Test/Mftf/Test/AdminAddImageToWYSIWYGProductTest.xml:
```diff
@@ -75,6 +75,7 @@
         <waitForLoadingMaskToDisappear stepKey="waitForLoading13"/>
         <see selector="{{ProductShortDescriptionWYSIWYGToolbarSection.CreateFolder}}" userInput="Create Folder" stepKey="seeCreateFolderBtn2" />
         <waitForLoadingMaskToDisappear stepKey="waitForLoading14"/>
+        <wait stepKey="wait" time="3"/>
```
Why we need it? and why it wasn't needed earlier?

drpayyne replied:
The test fails since there’s a delay so I added it.

ihor-sviziev reviewed (changes requested):
Hi @drpayyne,
Could you review and fix failing tests/adjust your code?

drpayyne commented:
@ihor-sviziev I have fixed the failing MFTF and integration test. Replaced the <wait/> with a better alternative and also fixed unnecessary code in the MFTF file.

lbajsarowicz reviewed (approved):
Nice job! Thanks for your contribution.

magento-engcom-team commented:
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-6168](https://jira.corp.magento.com/browse/ENGCOM-6168) has been created to process this Pull Request
:eight_spoked_asterisk: @lbajsarowicz, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

drpayyne commented:
@lbajsarowicz Thank _you_!

engcom-Alfa commented:
Hi @drpayyne 

During testing, we faced the issue

_**Problem:**_ __404 Not found_ after updating Wish List

**Steps to reproduce:**
1. Add simple product to Wishlist;
2. From Wish List click on edit :pen: product button;
3. Change, for example, the quantity of product;
4. Click on "Update Wish List" button;

**Actual Result:** _404 not found_ error occurs
![after](https://user-images.githubusercontent.com/51679138/67764284-4ac1fc00-fa52-11e9-9cf7-16f0fec6d7e5.png)

Probably, the problem is with _HttpGetActionInterface_ in _UpdateItemOptions_ class

@drpayyne Could you take a look?

Thanks!

drpayyne commented:
Hi @engcom-Alfa, thank you for your detailed comment. I've fixed it and sorry I missed that out. Thank you!

magento-engcom-team commented:
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-6168](https://jira.corp.magento.com/browse/ENGCOM-6168) has been created to process this Pull Request

engcom-Alfa commented:
:heavy_check_mark: QA Passed
